packages/node_modules/@webex/plugin-meetings/README.md

```diff
@@ -714,46 +922,308 @@ Using real time data pushes, you can add data to the chart by the interval
+```
+
+##### Functions
+```javascript
+// You can add a guest to the meeting by inviting them, this is proxied by meeting.invite
+// use an emailAddress and a boolean value alertIfActive to notify server side (usually true)
+meeting.members.addMember(emailAddress, alertIfActive)
+
+// You can admit the guest to the meeting once they are waiting in the lobby, you can do this in bulk, proxied by meeting.admit
+// use member ids, can be singular, but has to be put into an array
+meeting.members.admitMembers([memberIds])
+
+
+// You can remove a member from the meeting by booting them, this is proxied by meeting.remove
+// use a memberId string
+meeting.members.removeMember(memberId)
+
+// You can audio mute a member from the meeting by calling to mute them, this is proxied by meeting.mute
+// use a memberId string and a boolean to mute or not, default to true
+// mute them
+meeting.members.muteMember(memberId)
+
+// You can transfer the host role to another member in the meeting, this is proxied by meeting.transfer
+// use a memberId string and a moderator boolean to transfer or not, default to true
+meeting.members.transferHostToMember(memberId)
+```
+
+##### Events
+```javascript
+// members collection updated
+meeting.members.on('members:update', (payload) => {
+  const delta = payload.delta; // the changes to the members list
+  const full = payload.full; // the full members collection
+  const updated = delta.updated; // only the updates, includes removals, as they will have updated status and member properties
+  const added = delta.added; // added members to the meeting
+  Object.keys(full).forEach((key) => {
+    const member = full[key];
+    console.log(`Member: ... ${member.x}`);
+  });
+  Object.keys(updated).forEach((key) => {
+    const member = updated[key];
+    console.log(`Member Updated: ... ${member.x}`);
+  });
+  Object.keys(added).forEach((key) => {
+    const member = added[key];
+    console.log(`Member Added: ... ${member.x}`);
+  });
+});
+// content updates
+meeting.members.on('members:content:update', (payload) => {
+  console.log(`who started sharing: ${payload.activeContentSharingId};`);
+  console.log(`who stopped sharing: ${payload.endedContentSharingId};`);
+});
+// host updates
+meeting.members.on('members:host:update', (payload) => {
+  console.log(`who started hosting: ${payload.activeHostId};`);
+  console.log(`who stopped hosting: ${payload.endedHostId};`);
+})
+// self updates, not typically used
+meeting.members.on('members:self:update', (payload) => {
+  console.log(`active self id: ${payload.activeSelfId};`);
+  console.log(`ended self Id: ${payload.endedSelfId};`);
+})
+```
+
 ## Events
 
-There are several events submitted by this package that you can subscribe to.
+### Meetings Events
+```js
+webex.meetings.on(...)
+```
+| Event Name | Description |
+|---|---|
+| `meetings:ready` | Fired when the plugin has been instantiated and is ready for action! |
```

Comment:
@chrisadubois I think the next item up is to rename the strings for events. I will create a ticket for that and let you know before it is pushed so Cantina can update too.